### DISCUSSION TITLE
Why can't we add languages to new channels?

### DISCUSSION THREAD

**Discussion by Manipulativemachan** [iptv-org/community#99] [Q&A] [answered]
Since the language and category fields are empty, we can't use the playlists well. any fix to this?

**Comment by BellezaEmporium** [comment, marked as answer]
There's no fix, it simply moved to @iptv-org/database : https://github.com/iptv-org/database/blob/master/data/channels.csv

**Reply by Manipulativemachan** [reply]
Not able to open/download the file. Says file is too large to view.

**Reply by BellezaEmporium** [reply]
Use GitHub Desktop and download the whole project, look after the channels.csv file and edit whatever you need.

**Reply by Manipulativemachan** [reply]
> Use GitHub Desktop and download the whole project, look after the channels.csv file and edit whatever you need.

I did the editing but no change is reflected in the list. Is there anything else I have to do? I'm a noob to the github desktop app.

**Reply by freearhey** [reply]
Here are short instructions on how to edit channel description in the database: https://github.com/iptv-org/database/blob/master/CONTRIBUTING.md#how-to-add-a-channel-to-the-database-or-edit-its-description
